### ISSUE TITLE
Tab order on likert component is wrong

### ISSUE THREAD

**Issue by RonnyB71** [Altinn/app-frontend-react#1730]
### Description of the bug

Using a Jaws screen reader to answer questions using the likert component the focus is wrong when navigating back to the component from a validation message on a summary page (and possible when entering the component in the first place) making it very hard for users to actually complete the answer.

### Steps To Reproduce

1. On a page with a likert component, skip filling out the answer and navigate to the last page and try to submit
2. Click on the validation issue pointing back to the likert component which should be required
3. Observe that the focus is on the last radio button on the answer and not the headline and that is not possible to easily navigate back

### Additional Information
Potentially related to #1408

**Comment by RonnyB71** [comment]
Currently blocked waiting for JAWS software to test against.

**Comment by Febakke** [comment]
One theory I have is that the table elements might be giving JAWS and NVDA som trouble. 
1. This component is basicly a radiogroup. The semantics of the table is not important to the task that the user needs to do. Im not sure what is possible here. But if we could build this as a radiogroup and use CSS to build the "table" we would probably have less issues with screen readers. 
2. Or set an [presentation role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/presentation_role) on the table. We just need to make sure the radio elements dosent inherit this role. Might be worth a quick test.
